tests/integration/plugins/test_scons.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+import textwrap
+from pathlib import Path
+
+import yaml
+
+from craft_parts import LifecycleManager, Step
+
+
+def test_scons_plugin(new_dir):
+    """Test builds with the scons plugin"""
+    source_location = Path(__file__).parent / "test_scons"
+
+    parts_yaml = textwrap.dedent(
+        f"""
+        parts:
+          foo:
+            plugin: scons
+            source: {source_location}
+            scons-options:
+              - greeting=Hello
+              - person-name=craft-parts
+        """
+    )
+    parts = yaml.safe_load(parts_yaml)
+    lf = LifecycleManager(
+        parts, application_name="test_scons", cache_dir=new_dir, work_dir=new_dir
+    )
+    actions = lf.plan(Step.PRIME)
+
+    with lf.action_executor() as ctx:
+        ctx.execute(actions)
+
+    binary = Path(lf.project_info.prime_dir, "bin", "hello")
+    assert binary.is_file()
```

Comment:
Since the package installs the built artifact directly at the destination root instead of bindir, it should test for the presence there as well (at `prime/hello`).

Reply:
Updated the `SConstruct` file to build into `DESTDIR/bin`